bin/configs/spring-boot-api-response-examples.yaml

```diff
@@ -0,0 +1,13 @@
+generatorName: spring
+outputDir: samples/server/petstore/springboot-api-response-examples
```

Comment:
shall we also update the github work to test the new folders?

Reply:
Yes. Done :)